int32[] stance_indices
int32[] d_stance_indices
std_msgs/Float64MultiArray forces
std_msgs/Float64MultiArray stance_vectors